digitaltwins/service/src/service.json

```diff
@@ -1536,7 +1781,7 @@
       "post": {
         "summary": "Invoke a digital twin interface command.",
         "description": "Invoke a digital twin interface command.",
-        "operationId": "DigitalTwin_InvokeInterfaceCommand",
+        "operationId": "DigitalTwin_InvokeComponentCommand",
```

Comment:
>InvokeComponentCommand" [](start = 36, length = 23)

This says component yet the parameter name is interfaceName, same for the path above.

Reply:
Yes, that is what has been generated. The service team still could not clean up this mess. I specifically asked them to get rid of all the "interfaces" but they probably have a reason.

---
In reply to: [449329625](https://github.com/Azure/azure-iot-sdk-node/pull/836#discussion_r449329625) [](ancestors = 449329625)